### PR TITLE
Edits, first batch

### PR DESCRIPTION
**Pull request by martinthomson**
These are pretty big.  If you would prefer that I create a PR for each individual commit, that can be done.

It turns out that there were a lot of things I found needed work.  For instance, this completely redoes the 0-RTT advice.

### CODE REVIEW

**Review by britram — APPROVED** [review]
Aside from the keep-alive tweak and the idempotence change, these are all editorial and LGTM, thanks!

The keepalive tweak is also good.

Having looked at https://github.com/quicwg/base-drafts/pull/4394, the text here is a good applicability-appropriate view on what's there now. assuming discussion there converges to mostly the same thing, I'd be happy to merge this.

(in the future, no need to make tiny PRs, but splitting into "editorial-rollup" and "substantive editorial changes" would be nice to separate discussion, since the former shouldn't need much of it)

**Comment by martinthomson** [comment]
It did kinda snowball, for which I apologize.  I was trying to get something under your nose before I got swamped by meetings.  I am also on vacation next week.

I'm reading through and there is some really good stuff in the doc.  I have started to open issues where I don't think the fixes are so simple.

File: draft-ietf-quic-applicability.md [hunk 1]
```diff
@@ -163,46 +166,45 @@ and challenges for applications using QUIC.
...
-## Here There Be Dragons
+Application protocols that use 0-RTT require a profile that describes the types
+of information that can be safely sent. For HTTP, this profile is described in
+{{?HTTP-REPLAY=RFC8470}}.
 
-Retransmission or (malicious) replay of data contained in 0-RTT resumption
-packets could cause the server side to receive two copies of the same data.
-This is further described in {{?HTTP-RETRY=I-D.nottingham-httpbis-retry}}.
-Data sent during 0-RTT resumption also cannot benefit from perfect forward
-secrecy (PFS).
```
Comment (mirjak) [inline draft-ietf-quic-applicability.md:185]: The new text does not mention PFS anymore. Do we need to retain that?

**Review by mirjak — APPROVED** [review]
Changes look all ready good. Thanks!

I just have left one minor comment/question to double-check before we merge.

File: draft-ietf-quic-applicability.md (hunk 1 above)
Comment (martinthomson) [reply draft-ietf-quic-applicability.md:185]: Forward secrecy (or post-compromise security to be precise) can be provided for 0-RTT under some conditions, though they are quite unusual in practice.  Generally, we wouldn't assume PFS for 0-RTT, but you can't claim that it is impossible to achieve.

**Pull request merged**